bundle/manifests/patterns-operator.clusterserviceversion.yaml

```diff
@@ -21,13 +21,12 @@ metadata:
       ]
     capabilities: Basic Install
     categories: OpenShift Optional
-    containerImage: quay.io/hybridcloudpatterns/patterns-operator:0.0.17
-    createdAt: 2023-07-26 15:50:32
+    containerImage: quay.io/claudiol/patterns-operator:6.6.45
     description: An operator to deploy and manage architecture patterns from https://hybrid-cloud-patterns.io
-    operators.operatorframework.io/builder: operator-sdk-v1.30.0
+    operators.operatorframework.io/builder: operator-sdk-v1.22.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/hybrid-cloud-patterns/patterns-operator
-  name: patterns-operator.v0.0.17
+  name: patterns-operator.v6.6.45
```

Comment:
What's going on here?

Reply:
I committed these files and I really shouldn't have.  I will fix this.

Reply:
Fixed in latest commit.